src/5e-SRD-Proficiencies.json

```diff
@@ -64,99 +79,169 @@
+	"references": [{
+		"url": "/api/equipment/studded-leather",
+		"type": "equipment",
+		"name": "Studded Leather"
+	}]
+  }, {
 	"index": "hide",
 	"type": "Armor",
 	"name": "Hide",
 	"classes": [],
 	"races": [],
-	"url": "/api/proficiencies/hide"
-}, {
+	"url": "/api/proficiencies/hide",
+	"references": [{
+		"url": "/api/equipment/hide",
+		"type": "equipment",
+		"name": "Hide"
+	}]
+  }, {
 	"index": "chain-shirt",
 	"type": "Armor",
 	"name": "Chain Shirt",
 	"classes": [],
 	"races": [],
-	"url": "/api/proficiencies/chain-shirt"
-}, {
+	"url": "/api/proficiencies/chain-shirt",
+	"references": [{
+		"url": "/api/equipment/chain-shirt",
+		"type": "equipment",
+		"name": "Chain Shirt"
+	}]
+  }, {
 	"index": "scale-mail",
 	"type": "Armor",
 	"name": "Scale Mail",
 	"classes": [],
 	"races": [],
-	"url": "/api/proficiencies/scale-mail"
-}, {
+	"url": "/api/proficiencies/scale-mail",
+	"references": [{
+		"url": "/api/equipment/scale-mail",
+		"type": "equipment",
+		"name": "Scale Mail"
+	}]
+  }, {
 	"index": "breastplate",
 	"type": "Armor",
 	"name": "Breastplate",
 	"classes": [],
 	"races": [],
-	"url": "/api/proficiencies/breastplate"
-}, {
+	"url": "/api/proficiencies/breastplate",
+	"references": [{
+		"url": "/api/equipment/breastplate",
+		"type": "equipment",
+		"name": "Breastplate"
+	}]
+  }, {
 	"index": "half-plate",
 	"type": "Armor",
 	"name": "Half Plate",
 	"classes": [],
 	"races": [],
-	"url": "/api/proficiencies/half-plate"
-}, {
+	"url": "/api/proficiencies/half-plate",
+	"references": [{
+		"url": "/api/equipment/half-plate",
+		"type": "equipment",
+		"name": "Half Plate"
+	}]
+  }, {
 	"index": "ring-mail",
 	"type": "Armor",
 	"name": "Ring Mail",
 	"classes": [],
 	"races": [],
-	"url": "/api/proficiencies/ring-mail"
-}, {
+	"url": "/api/proficiencies/ring-mail",
+	"references": [{
+		"url": "/api/equipment/ring-mail",
+		"type": "equipment",
+		"name": "Ring Mail"
+	}]
+  }, {
 	"index": "chain-mail",
 	"type": "Armor",
 	"name": "Chain Mail",
 	"classes": [],
 	"races": [],
-	"url": "/api/proficiencies/chain-mail"
-}, {
+	"url": "/api/proficiencies/chain-mail",
+	"references": [{
+		"url": "/api/equipment/chain-mail",
+		"type": "equipment",
+		"name": "Chain Mail"
+	}]
+  }, {
 	"index": "splint",
 	"type": "Armor",
 	"name": "Splint",
 	"classes": [],
 	"races": [],
-	"url": "/api/proficiencies/splint"
-}, {
+	"url": "/api/proficiencies/splint",
+	"references": [{
+		"url": "/api/equipment/splint",
+		"type": "equipment",
+		"name": "Splint"
+	}]
+  }, {
 	"index": "plate",
 	"type": "Armor",
 	"name": "Plate",
 	"classes": [],
 	"races": [],
-	"url": "/api/proficiencies/plate"
-}, {
+	"url": "/api/proficiencies/plate",
+	"references": [{
+		"url": "/api/equipment/plate",
+		"type": "equipment",
+		"name": "Plate"
+	}]
+  }, {
 	"index": "shield",
 	"type": "Armor",
 	"name": "Shield",
 	"classes": [],
 	"races": [],
-	"url": "/api/proficiencies/shield"
-}, {
+	"url": "/api/proficiencies/shield",
```

Comment:
It seems weird that there is both `Shield` and `Shields`? Should these be actually combined instead?

Reply:
This is a symptom of the data.  In dnd5 there is only one shield, but the APi exposes both a shield equipment resource and a shields equipment-categories resource. Since proficiencies can be on specific items, or on equipment categories both proficiencies are required here.

Moreover, all of the classes point at "shields" even though I think they should point at shield.

I can replace all references to /api/equipment-categories/shield to /api/equipment/shields and remove the shields equipment-category

Reply:
Ahhhhh! But all the weapon proficiencies are plural. I think I'll leave the proficiency plural, point at the singular equipment item, and get rid of the shields equipment category